src/data/projects.json

```diff
@@ -173,7 +173,7 @@
         "bullets": [
             "We helped put Oppia users in control of their own data.",
             "We improved the development setup experience for future Oppia contributors across the globe.",
-            "By writing and maintaining Oppia's end-to-end tests, we helped keep Oppia a stable platform for teachers, learners, and developers."
+            "By writing and maintaining Oppqqia's end-to-end tests, we helped keep Oppia a stable platform for teachers, learners, and developers."
```

Comment:
Not sure why this was added, but it seems like a typo

Reply:
lol